pkgs/development/libraries/science/math/tiny-cuda-nn/default.nix

```diff
@@ -0,0 +1,156 @@
+{
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  symlinkJoin,
+  which,
+  pythonPackages,
+  pythonSupport ? false,
+}:
+assert lib.asserts.assertMsg
+(pythonPackages.torch.cudaSupport)
+"tiny-cuda-nn requires torch to be built with cudaSupport"; let
```

Comment:
hmm this formatting seems a bit wonky... could you please hit this whole file with nixpkgs-fmt or whatever your formatter of choice is?

Reply:
Sorry, that was part Alejandra and part me forgetting to run the formatter again.